cli/replicaset/lua/cartridge/edit_topology_body.lua

```diff
@@ -0,0 +1,18 @@
+local cartridge = require('cartridge')
+
+local args, wait_rw, timeout = ...
+
+local res, err = cartridge.admin_edit_topology({
+    servers = args.servers,
+    replicasets = args.replicasets
+})
```

Comment:
Could `servers` and `replicasets` be used both? Or it is two different methods in fact?

https://github.com/tarantool/cartridge-cli/blob/76044114f412b1fa15e353f84e7de1f0c3d98566/cli/replicasets/lua/edit_instance_body.lua
https://github.com/tarantool/cartridge-cli/blob/76044114f412b1fa15e353f84e7de1f0c3d98566/cli/replicasets/lua/edit_replicasets_body_template.lua

Reply:
We can specify "servers" or "replicasets" in the request argument, as we want, so this file is the `edit_topology` wrapper. Why we need to separate?

Reply:
Because in fact these are two different methods with different goals. It may be confusing.

Reply:
Thank you! I extracted the version with `replicasets` as `edit_replicasets_body`.